addons/account/models/chart_template.py

```diff
@@ -374,8 +374,10 @@ def _pre_load_data(self, template_code, company, template_data, data):
             and key in company._fields
         )
 
+        filter_not_defined_yet = lambda key: key in company._fields or self.env.context.get('l10n_check_fields_complete')
+
         # Set the currency to the fiscal country's currency
-        vals = {key: val for key, val in template_data.items() if filter_properties(key)}
+        vals = {key: val for key, val in template_data.items() if filter_properties(key) and filter_not_defined_yet(key)}
```

Comment:
This is useless, there is no field declared directly in the `template_data` (only properties and some metadata)